.goreleaser.yaml

```
@@ -9,6 +9,7 @@
 version: 1
 before:
   hooks:
+    - if [ "$(git describe --tags --exact-match 2> /dev/null)" = "" ]; then git tag $(git describe --tags --abbrev=0 | awk -F. '{$NF = $NF + 1;} 1' | sed 's/ /./g'); fi
```

chatGPTに書いてもらった。
tagがなければ、一つ前のtagからパッチバージョンを1足して新しくtagをつけてくれそう。

---

```suggestion
    - [ "$(git describe --tags --exact-match 2> /dev/null)" = "" ] && git tag $(git describe --tags --abbrev=0 | awk -F. '{$NF = $NF + 1;} 1' | sed 's/ /./g')
```

ワンライナーならこちらの方がよいかもしれない？

---

いいね

---

これ本当に動く？
なんかローカルで上手く動かなさそうな気配を感じたので、一旦戻します。